proto/public/api.proto

```diff
@@ -159,6 +159,8 @@ message BasicStats {
   uint64 latency_ms_p50 = 3;
   uint64 latency_ms_p95 = 4;
   uint64 latency_ms_p99 = 5;
+  // count of requests that are TLSed
+  uint64 secured_request_count = 6;
```

Comment:
this may be ok for now to satisfy the immediate cli/web use cases, but want to note that we're losing a bit of information here, for example, the metrics in prometheus will give us secured requests broken down by success/failure, so we'd end up with something more like:
```
uint64 secured_success_count = 6;
uint64 secured_failure_count = 7;
```

Reply:
Yep! this is true. I've collapsed all meshed requests into one count, as we only care about the total right now. I'm open to splitting this out though.